crates/stdarch-gen/src/main.rs

```diff
@@ -1064,9 +1064,9 @@ fn gen_aarch64(
     };
     let current_target = match target {
         Default => "neon",
-        ArmV7 => "v7",
-        Vfp4 => "vfp4",
-        FPArmV8 => "fp-armv8,v8",
+        ArmV7 => "neon",
+        Vfp4 => "neon",
+        FPArmV8 => "neon",
```

Comment:
Why did you change this? The previous target features were correct.

Reply:
There are two functions used to generate instructions: `gen_arm` and `gen_aarch64`. The part I changed is in `gen_aarch64`. The values of these targets do not affect the `target_feature` on aarch64, so unified to `neon` I think it is more appropriate

Reply:
That's fine if it's only on aarch64.